qcengine/programs/dftd3/runner.py

```diff
@@ -291,14 +293,14 @@ def dftd3_harvest(jobrec, modulerec):
         elif re.match(' Edisp /kcal,au', ln):
             ene = Decimal(ln.split()[3])
         elif re.match(r" E6\(ABC\) \"   :", ln):  # c. v3.2.0
-            raise ValueError("Cannot process ATM results from DFTD3 prior to v3.2.1.")
+            raise ResourceError("Cannot process ATM results from DFTD3 prior to v3.2.1.")
         elif re.match(r""" E6\(ABC\) /kcal,au:""", ln):
             atm = Decimal(ln.split()[-1])
         elif re.match(' normal termination of dftd3', ln):
             break
     else:
         if not ((real_nat == 1) and (jobrec['driver'] == 'gradient')):
-            raise ValueError('Unsuccessful run. Possibly -D variant not available in dftd3 version.')
+            raise ResourceError('Unsuccessful run. Possibly -D variant not available in dftd3 version.')
```

Comment:
The "-D variant not available" is only a minor chance since been available since 2016. So probably r`aise UnknownError('Unsuccessful gradient collection.')` like below. But that can be patched up later.